### DISCUSSION TITLE
Replace required readonly modifier with syncobject field

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#203] [Feature Request]
For the purpose of enabling Unity to serialze SyncObjects in the inspector, the readonly modifier would have to removed. This is not currently possible because FishNet requires the readonly modifier to protect against user errors, but it has been mentioned by @FirstGearGames that an alternative is possible:
> I can add a field to Sync object, EnforceReadonly or something

**Comment by FirstGearGames** [comment]
Added in 3.1. Made this document page which shows it better.
https://fish-networking.gitbook.io/docs/manual/guides/synchronizing/attributes
